src/Root.jsx

```diff
@@ -7,18 +7,6 @@ import { Web3ContextProvider } from "./hooks/web3Context";
 import App from "./App";
 import store from "./store";
 
-// Google Analytics
-import ReactGA from "react-ga4";
-const TRACKING_ID = "G-TKEFGWGLPM";
-ReactGA.initialize(TRACKING_ID);
-//
-// Initialize google analytics page view tracking
-import { createBrowserHistory } from "history";
-const gaHistory = createBrowserHistory();
-gaHistory.listen(location => {
-  ReactGA.send({ hitType: "pageview", page: location.pathname });
-});
-
```

Comment:
none of the above works properly with `BrowserRouter`. And with `BrowserRouter basename={"/#"}` react-ga4 thinks that window.location.pathname is ALWAYS "/", which is why GA wasn't picking up the page views.